packages/nocodb/src/helpers/dataHelpers.ts

```diff
@@ -165,7 +165,9 @@ export async function serializeCellValue(
         if (typeof value === 'string') {
           data = JSON.parse(value);
         }
-      } catch {}
+      } catch {
```

Comment:
We can add an additional check to confirm if it's an array or not.

Reply:
> We can add an additional check to confirm if it's an array or not.

Okay, I've added the array check.